Command/CleanMediaCommand.php

```diff
@@ -121,14 +96,11 @@ private function mediaExists($filename, $context = null)
 
         if (count($fileParts) > 1 && $fileParts[0] == 'thumb') {
             return $mediaManager->findOneBy(array(
-                    'id' => $fileParts[1],
-                    'context' => $context,
-                )) != null;
+                'id' => $fileParts[1],
+                'context' => $context,
+            )) != null;
         }
 
-        return count($mediaManager->findBy(array(
-                'providerReference' => $filename,
-                'providers' => $this->getProviders(),
```

Comment:
Not really sure, this was implemented two years ago. https://github.com/sonata-project/SonataMediaBundle/pull/898

One year ago some guy just discover the bug, but the PR was never created.

I did look at the orm definition, but couldn't see any `providers` column (looking at all the modification of the columns)